### DISCUSSION TITLE
how to pull down the page

### DISCUSSION THREAD

**Discussion by legendred999** [AutomaApp/automa#622] [General]
I don't know how to pull down the page ，i try to use the Scroll Element Block .but it doesn't work. is that other element can do it

**Comment by Kholid060** [comment]
What do you mean by "pull down the page"? Can you show me the website?

**Reply by mzaini30** [reply]
I think it's like pull down for reload page
https://stackoverflow.com/questions/30071319/how-to-make-pull-down-page-refresh-on-web-app-using-jquery-mobile-or-javascript

**Reply by SuperTempAccount** [reply]
Would reloading the tab achieve the same thing?
Use a `Reload Tab` block.

If the OP is referring to scrolling down, then a `Scroll element` block can be used for that.
